package.json

```diff
@@ -32,27 +32,21 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "axios": "^0.26.1",
-    "chai": "^4.2.0",
+    "chai": "^4.3.6",
     "chai-as-promised": "*",
     "chalk": "^2.4.2",
-    "cross-env": "^5.2.0",
     "cypress": "^9.5.3",
     "if-env": "^1.0.4",
     "jest": "^27.5.1",
-    "mocha": "^9.1.0",
+    "mocha": "^9.2.2",
     "recursive-readdir": "^2.2.2",
     "remove": "^0.1.5"
   },
   "dependencies": {
     "broken-link-checker": "^0.7.8",
-    "hoek": "^6.1.3",
-    "joi": "^14.3.1",
-    "serve": "^11.0.0",
+    "serve-handler": "^6.1.3",
     "vuepress": "1.9.7",
     "vuepress-plugin-chunkload-redirect": "^1.0.3",
     "wait-for-localhost-cli": "^3.0.0"
```

Comment:
I think most of these can be moved to `devDependencies`, no?

e.g. `serve`/`serve-handler` (and `broken-link-checker`) is used by the broken link checker script

I think the only ones here that might need to stay in `dependencies` is the `vuepress` and `vuepress-plugin-*`